src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.StorageFruit;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruitName, int amountOfFruit) {
+        StorageFruit.storage.put(fruitName, amountOfFruit);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return StorageFruit.storage.get(fruit);
```

Handle the case where the fruit might not exist in the storage to prevent NullPointerException. You can return a default value or throw an exception if the fruit is not found.